packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -209,6 +209,9 @@ export default compose(
 		const shouldAddClipboardBlock =
 			clipboardBlock &&
 			canInsertBlockType( clipboardBlock.name, destinationRootClientId );
+		const inserterItems = getInserterItems(
+			destinationRootClientId
+		).filter( ( { name } ) => name !== 'core/block' );
```

Comment:
When the reusable block type is registered (`core/block` type), all reusable block items are added into the inserter menu mixed with the rest. The idea is to have them in a separate tab so I'm filtering them out.